xml/obs_ag_user_management.xml

```diff
@@ -107,6 +107,14 @@
    confirmation is needed after registration before the user may
    login.</para>
  </sect2>
+ <sect2 xml:id="_users_and_group_maintainers">
+  <title>Users and Group Maintainers</title>
+  <para>Administrators can create new groups and add and remove users on it, as
+    well as give the users Maintainer rights. This way, a mantainer user will
```

Comment:
Administrators can create <strike>new</strike> groups, add users to them , remove users from them and give Maintainer rights to users. This way, a ma**i**ntainer will (...)